src/qibosoq/programs/sweepers.py

```diff
@@ -1,11 +1,11 @@
 """Program used by qibosoq to execute sweeps."""
 
 import logging
-from typing import Iterable, List, Tuple, Union
+from typing import Iterable, List, Union
 
 import numpy as np
-from qick import NDAveragerProgram, QickSoc
-from qick.averager_program import QickSweep, merge_sweeps
+from qick import NDAveragerProgram, QickSoc  # type: ignore
+from qick.averager_program import QickSweep, merge_sweeps  # type: ignore
```

Comment:
If the problem is `qick` not being properly typed, we could whitelist it in the `mypy` configs.

Reply:
Ok, it makes sense